workers-docs/src/content/reference/apis/web-crypto.md

```diff
@@ -33,7 +33,7 @@ We are happy to add support for more algorithms – [let us know](https://commun
 | SHA-256           |                    |                        | ✓        |                             |               |                          |
 | SHA-384           |                    |                        | ✓        |                             |               |                          |
 | SHA-512           |                    |                        | ✓        |                             |               |                          |
-| MD5\*<sup>1</sup>   |                    |                        | ✓        |                             |               |                          |
+| MD5\*             |                    |                        | ✓        |                             |               |                          |
```

Comment:
as an aside, I think this `\*` here was introduced accidentally in a previous PR and we should get rid of it.